std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -168,12 +168,12 @@ public static Column fromItemsNoDateConversion(String name, List<Object> items,
    * @return a column with given name and items
    */
   public static Column fromRepeatedItems(String name, List<Value> items, int repeat, ProblemAggregator problemAggregator) {
-    if (repeat < 1) {
-      throw new IllegalArgumentException("Repeat count must be positive.");
-    }
-
-    if (repeat == 1) {
+    if (repeat < 0) {
+      throw new IllegalArgumentException("Repeat count must be non-negative.");
+    } else if (repeat == 1) {
       return fromItems(name, items, null, problemAggregator);
+    } else if (repeat == 0) {
+      return new Column(name, new InferredBuilder(0, problemAggregator).seal());
```

Comment:
With this approach, I assume this will always create a column of the same type. 

As we have discussed, we need the type to be consistent regardless of the `repeat` value.

Reply:
won't this always be a `Mixed` column and we need it to get the correct type?

Reply:
Done